### ISSUE TITLE
Update form button and auto-save conflict in forms

### ISSUE THREAD

**Issue by mrblanco1** [department-of-veterans-affairs/va.gov-team#35723]
### User story(ies)

As a Veteran when I am filling out forms  (i.e. form 10203 or 1990 - we suspect this is a global issue across all forms, and I make a change without clicking the Update form button and then submitting the form, the changes are auto-saved. This causes issues since I might have thought that since the Update form button was NOT clicked changes were not saved, when in reality they were saved.

The auto-save and Update form button conflict and mislead Veterans. 

My question is, do we want to preserve the Auto Save and remove all Update form buttons? Or do we want to remove the auto save and preserve the update form buttons?

We are working on this ticket:
app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/34408
What would your recommendation be?

### Acceptance criteria

AFS and EDU comparison tool are requesting OCTO's Design team for guidance on this issue. Please advise.

### Point(s) of contact

Abe Blanco abelardo.blanco@accenturefederal.com

### What type of issue is this? 
- [X] Bug
- [X] Request update to existing feature

I am submitting this ticket per Andrés Rivera Toro's request.

### How to configure this issue
- [ ] When ticket is opened, assign to Epic [#12765 Forms library user feedback and requests to be triaged](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/12765)
- [ ] FE Tools team will triage and add to [Design System Governance Process Project Board](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/projects/3) as applicable

**Comment by k80bowman** [comment]
@caw310 we may want to talk with the DSC about this. What do you think?

**Comment by pdavies88** [comment]
@k80bowman @caw310 for reference - team slack convo: https://dsva.slack.com/archives/G01BJ3ESXL4/p1642714270305400

**Comment by k80bowman** [comment]
![image](https://user-images.githubusercontent.com/12739849/151600087-4282a72c-9925-4320-afe5-6aa350157e1e.png)

**Comment by pdavies88** [comment]
Summary of the issue at hand:
1. On forms with Save in Progress enabled, Save in Progress works through the full form even the `review-and-submit` page
2. Due to Save in Progress working on the `review-and-submit` page when a user clicks the Accordion and Edit button when they make changes to a field those changes are saved without any indication to let the user know
3. Additionally a User is receiving conflicting feedback on how to make changes and save their information because we also have an Update Page button on those forms that will also save changes
4. If a User makes changes and wasn't intending on having those changes saved there is no way for user to undo their changes as Save In Progress would have already made the updates

**Comment by mrblanco1** [comment]
Hello @pdavies88 - Good afternoon/evening. Thank you for looking at this issue. I was wondering if there are already steps in place and timelines to correct this behavior. I am asking because I wonder if we should close issue ##34408 so this effort can be driven through this ticket.
Thank you!

**Comment by pdavies88** [comment]
@mrblanco1 I would agree we probably don't need 2 tickets for the same issue we could probably just track it moving forward via this one and make sure the other ticket is notated that this ticket will be the one worked on to resolve whatever decision the Design System Council makes in the future about this situation

**Comment by mrblanco1** [comment]
> @mrblanco1 I would agree we probably don't need 2 tickets for the same issue we could probably just track it moving forward via this one and make sure the other ticket is notated that this ticket will be the one worked on to resolve whatever decision the Design System Council makes in the future about this situation

Thank you @pdavies88 - I will close #34408 and keep #35723 open. Do you have a contact name EDU can reach with the Design System Council team/group or can you tell us how we can connect with them? Thanks!

**Comment by pdavies88** [comment]
> > @mrblanco1 I would agree we probably don't need 2 tickets for the same issue we could probably just track it moving forward via this one and make sure the other ticket is notated that this ticket will be the one worked on to resolve whatever decision the Design System Council makes in the future about this situation
> 
> Thank you @pdavies88 - I will close #34408 and keep #35723 open. Do you have a contact name EDU can reach with the Design System Council team/group or can you tell us how we can connect with them? Thanks!

@mrblanco1 I think @k80bowman and @caw310 would be your best resources to reach out to

**Comment by k80bowman** [comment]
@mrblanco1 we discussed this issue with the Design System Council on January 28th. I apologize for not adding a comment to this ticket earlier. This issue is more complex than it seems on the surface and will require some user research to determine the best path forward. The DSC has flagged this issue as a future improvement to the review page.

**Comment by caw310** [comment]
@mrblanco1 Is there an update on the user research around this issue?

**Comment by KevinMHoffmanUSDS** [comment]
DSC determined this as medium, we should assess if recent or near term review page research provides guidance on this issue. @biannuz

**Comment by taylorkaren** [comment]
The new Review Page pattern to be implemented in the VA Forms System Core (VAFSC) will not use the accordion pattern with inline editing on the Review Page itself. See the [Round Two Review Page Research Findings Confluence page](https://vfs.atlassian.net/wiki/spaces/FLT/pages/2309324815/Round+Two+Review+Page+Research+Findings) for a link to a slide deck with screen designs for the new pattern.

This new pattern will navigate form fillers back to the appropriate page in the form to make changes and will provide a clearly labeled button (e.g., Save and continue) which will be the one way to indicate that a change to the previously entered form data should be saved.

With the current Forms Library being sunsetted, addressing this issue will not be prioritized. The focus of the Forms Library Team's work is on building out a better review page pattern in the new VAFSC package.

**Comment by caw310** [comment]
@tbaker1026 I think this is now a form issue.
